src/GitHub.InlineReviews/CommentsMargin.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.IO;
+using System.Windows;
+using System.Threading.Tasks;
+using System.Reactive.Linq;
+using GitHub.Models;
+using GitHub.Commands;
+using GitHub.Services;
+using GitHub.Extensions;
+using GitHub.InlineReviews.Views;
+using GitHub.InlineReviews.ViewModels;
+using GitHub.InlineReviews.Commands;
+using Microsoft.VisualStudio.Text.Editor;
+using ReactiveUI;
+using Task = System.Threading.Tasks.Task;
+
+namespace GitHub.InlineReviews
+{
+    internal class CommentsMargin : IWpfTextViewMargin
```

Comment:
I need to find a better name of this margin. It's the horizontal margin that currently appears when editing a PR file in the working directory.

Ultimately it could give more information about the PR the user is working on and include commands to jump to the next PR file.

Maybe `PullRequestMargin` would be a better name? It will only be visible when working on a checked out pull request.